plugin/storage/cassandra/schema/docker.sh

```diff
@@ -3,7 +3,7 @@
 # This script is used in the Docker image jaegertracing/jaeger-cassandra-schema
 # that allows installing Jaeger keyspace and schema without installing cqlsh.
 
-CQLSH=/opt/apache-cassandra-3.0.12/bin/cqlsh
+CQLSH=/usr/bin/cqlsh
```

Comment:
I don't think making the env var would have been a great time saver for a change like this. But in any case, I'll make it use an env var, defaulting to `/usr/bin/cqlsh`

About the other proposed changes, they are not the scope of this change. Perhaps this could be done in a follow up PR, but the script was done to work strictly within a Docker image, and I have not tested it in any other situation.